tests/unit/test_authorization_manager.py

```diff
@@ -217,9 +257,115 @@ def test_check_token(self, m):
     def test_check_token_with_no_token(self, m):
         auth = AuthorizationManager(identity_url='http://fake_url', api_version=AUTH_API_V2)
 
-        m.get('http://fake_url/tokens/token', text=self.validate_info_v2)
+        m.get('http://fake_url/tokens/token', json=self.validate_info_v2)
 
         try:
-            auth.checkToken(admin_token='admin_token', token='token')
+            auth.check_token(admin_token='admin_token', token='token')
         except Unauthorized as e:
             self.assertEquals(e.message, "Token is empty", 'The expected auth token is not the same')
+
+    def test_is_admin_should_return_false_without_user_no_admin(self, m):
+        # Given
+        auth = AuthorizationManager(identity_url='http://fake_url', api_version=AUTH_API_V2)
+        roles_list = [
+            {u'id': u'8d27cbfdaf3845b8a5cfc349f0b52bac', u'name': u'owner'},
+            {u'is_default': True, u'id': u'a6c6f50bc3ff438ab311a9063610d383', u'name': u'other'}]
+
+        # When
+        result = auth._is_admin(roles_list)
+        # Then
+        self.assertFalse(result)
+
+    def test_is_admin_should_return_true_with_admin_user(self, m):
+        # Given
+        auth = AuthorizationManager(identity_url='http://fake_url', api_version=AUTH_API_V2)
+        roles_list = [
+            {u'id': u'8d27cbfdaf3845b8a5cfc349f0b52bac', u'name': u'owner'},
+            {u'is_default': True, u'id': u'a6c6f50bc3ff438ab311a9063610d383', u'name': u'admin'}]
+
+        # When
+        result = auth._is_admin(roles_list)
+        # Then
+        self.assertTrue(result)
+
+    def test_is_admin_should_return_false_with_empty_roles(self, m):
+        # Given
+        auth = AuthorizationManager(identity_url='http://fake_url', api_version=AUTH_API_V2)
+        roles_list = None
+        # When
+        result = auth._is_admin(roles_list)
+        # Then
+        self.assertFalse(result)
+
+    def test_raise_exception_with_user_not_admin_with_from_keystone_v3(self, m):
+        # Given
```

Comment:
It seems that something is missing in this title:

test_raise_exception_with_user_not_admin_**with_from**_keystone_v3